.gitignore

```diff
@@ -4,3 +4,7 @@ bin
 
 ## Intermediate build files
 build
+
+
+## Swap Files
+*.swp
```

Comment:
Are swap files related to Vim? If so then they get deleted automatically after you save and/or exit from the editor, and so they don't need to be added to gitignore.